sys/net/gnrc/network_layer/ipv6/nib/nib_abr.c

```diff
@@ -50,8 +55,39 @@ void gnrc_ipv6_nib_abr_del(const ipv6_addr_t *addr)
     _nib_abr_remove(addr);
     mutex_unlock(&_nib_mutex);
 }
+#endif  /* GNRC_IPV6_NIB_CONF_6LBR */
+
+bool gnrc_ipv6_nib_abr_iter(void **state, gnrc_ipv6_nib_abr_t *entry)
+{
+    _nib_abr_entry_t *abr = *state;
+
+    mutex_lock(&_nib_mutex);
+    while ((abr = _nib_abr_iter(abr)) != NULL) {
+        if (!ipv6_addr_is_unspecified(&abr->addr)) {
+            memcpy(&entry->addr, &abr->addr, sizeof(entry->addr));
+            entry->version = abr->version;
+            entry->valid_until = abr->valid_until;
+            break;
+        }
+    }
+    mutex_unlock(&_nib_mutex);
+    *state = abr;
+    return (*state != NULL);
+}
+
+void gnrc_ipv6_nib_abr_print(gnrc_ipv6_nib_abr_t *abr)
+{
+    char addr_str[IPV6_ADDR_MAX_STR_LEN];
+    uint32_t now = _now_min();
+
+    printf("%s v%" PRIu32 " expires %" PRIu32 "min\n",
```

Comment:
```suggestion
    printf("%s v%" PRIu32 " expires %" PRIu32 " min\n",
```

looks nicer

Reply:
Matter of taste + the other print functions in `gnrc_ipv6_nib` use the same formatting.

See e.g.

https://github.com/RIOT-OS/RIOT/blob/729ba07e9acb4c84a411f3da1ba034a8cd125317/sys/net/gnrc/network_layer/ipv6/nib/nib_pl.c#L151-L156

If you want it to be changed, I gladly ACK a follow-up of your's that changes all of them :->